File: SwissO.Droid/ListFragmentPagerAdapter.cs
using AndroidX.Fragment.App;
using AndroidX.ViewPager2.Adapter;
using System.Collections.Generic;

namespace SwissO.Droid {
    public class ListFragmentPagerAdapter : FragmentStateAdapter {

        private ListFragment parent;
        private ListManager listManager;

        private List<SubListFragment> subListFragments = new List<SubListFragment>();

        public ListFragmentPagerAdapter(ListFragment fragment, ListManager listManager) : base(fragment) {
            this.listManager = listManager;
            parent = fragment;
        }

        public override int ItemCount => 3;

        public override Fragment CreateFragment(int position) {
            SubListFragment.ListContent listContent = SubListFragment.ListContent.alle;
            switch (position) {
                case 0:
                    listContent = SubListFragment.ListContent.Friends;
                    break;
                case 1:
                    listContent = SubListFragment.ListContent.Club;
                    break;
                case 2:
                    listContent = SubListFragment.ListContent.alle;
                    break;
            }
            SubListFragment subListFragment = new SubListFragment(listManager, listContent);
            subListFragments.Add(subListFragment);
            return subListFragment;
        }

        public void UpdateList() {
            foreach (SubListFragment fragment in subListFragments) {
                fragment.LoadList();
            }

        }
    }
}

File: app/src/main/java/ch/laasch/swisso/ListFragmentPagerAdapter.java
package ch.laasch.swisso;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.viewpager2.adapter.FragmentStateAdapter;

import java.util.ArrayList;

public class ListFragmentPagerAdapter extends FragmentStateAdapter {

    private final ListFragment listFragment;

    private final SingleListFragment[] singleListFragments = new SingleListFragment[3];

    public ListFragmentPagerAdapter(@NonNull ListFragment fragment) {
        super(fragment);
        listFragment = fragment;
    }

    @NonNull
    @Override
    public Fragment createFragment(int position) {
        SingleListFragment.ListContent listContent;
        switch (position) {
            case 0:
                listContent = SingleListFragment.ListContent.Friends;
                break;
            case 1:
                listContent = SingleListFragment.ListContent.Club;
                break;
            case 2:
            default:
                listContent = SingleListFragment.ListContent.alle;
                break;
        }
        SingleListFragment subListFragment = new SingleListFragment(listFragment, listContent);
        singleListFragments[position] = subListFragment;
        return subListFragment;
    }

    public void updateList() {
        for (SingleListFragment fragment : singleListFragments) {
            if (fragment != null) {
                fragment.loadList();
            }
        }
    }

    @Override
    public int getItemCount() {
        return 3;
    }

    public void setRefreshing(boolean b, int position) {
        if (singleListFragments[position] != null) {
            singleListFragments[position].setRefreshing(b);
        }
    }

    public void toggleSearch(int position) {
        if (singleListFragments[position] != null) {
            singleListFragments[position].toggleSearch();
        }
    }
}
